config.yaml

```diff
@@ -454,3 +458,36 @@ commands:
         - xcache
         - Zend Optimizer
         - Zend OPcache
+
+    env:
+      required-keys:
+        - backend.frontName
+        - crypt.key
+        - db.table_prefix
+        - db.connection.default.host
+        - db.connection.default.dbname
+        - db.connection.default.username
+        - db.connection.default.password
+        - db.connection.default.model
+        - db.connection.default.engine
+        - db.connection.default.initStatements
+        - db.connection.default.active
+        - resource.default_setup.connection
+        - x-frame-options
+        - MAGE_MODE
+        - session.save
+        - cache_types.config
+        - cache_types.layout
+        - cache_types.block_html
+        - cache_types.collections
+        - cache_types.reflection
+        - cache_types.db_ddl
+        - cache_types.eav
+        - cache_types.customer_notification
+        - cache_types.config_integration
+        - cache_types.config_integration_api
+        - cache_types.full_page
+        - cache_types.translate
+        - cache_types.config_webservice
+        - cache_types.compiled_config
+        - install.date
```

Comment:
I've seen you've been taking care of white-space. Please note as well the missing Newline at the end of file here.

Reply:
@ktomk Thanks for the hint. Will check that before we release the next version.

Reply:
@ktomk Checked that. Seems that for YAML files it's unusal to add a newline at the end. But I did not find a hint in the official specification.

Reply:
@cmuench YAML has no such requirement, it's just POSIX or UNIX text files which aligns very well with YAML. Also git normally highlights those missing, nowadays Github, too.